File: RB101_RB109_small_problems/easy_2/when_will_i_retire.rb
puts "What is your age (in years)?"
current_age = gets.strip.chomp.to_i

puts "At what age (in years) would you like to retire?"
retirement_age = gets.strip.chomp.to_i

current_year = Time.now.year
working_years_left = retirement_age - current_age
retirement_year = current_year + working_years_left

puts "It's #{current_year}. You will retire in #{retirement_year}."
puts "You only have #{working_years_left} years of work to go!"

File: JS210_small_problems/easy_3/when_will_i_retire.js
let rlSync = require('readline-sync');

let age = Number.parseInt(rlSync.question('What is your age? '), 10);
let retirementAge = Number.parseInt(rlSync.question('At what age would you like to retire? '), 10);

let currentYear = new Date().getFullYear();
let retirementYear = currentYear + (retirementAge - age);

console.log(`It's ${currentYear}. You will retire in ${retirementYear}.`);
console.log(`You have only ${retirementYear - currentYear} years of work to go!`);